src/shared/lib/oql/oql-parser.pegjs

```diff
@@ -19,7 +19,7 @@ zmbs = zero_or_more_breaks_and_spaces:[; \t\r\n]*
 ombs = one_or_more_breaks_and_spaces:[; \t\r\n]+
 
 StartMergedGenes
-	= "[" zmbs "\"" StringExceptQuotes "\"" {return {"label": label, "list":[]};}
+	= "[" zmbs "\"" label:StringExceptQuotes "\"" {return {"label": label, "list":[]};}
```

Comment:
this bug was introduced before. @inodb seems like oql-parser.spec.ts isn't running with the build?